src/presentation/meshtenant-iam-table-view.ts

```diff
@@ -10,6 +10,7 @@ export class MeshTenantIamTableViewGenerator extends TableGenerator {
     readonly meshTenants: MeshTenant[],
     readonly columns: (keyof MeshTenant)[],
     readonly includeAncestors: boolean,
+    readonly userFilterString?: string,
```

Comment:
d: we must be careful that such flags don't lead to spaghetti code. On the other hand I acknowledge that this is a simple case where this just does what it's supposed to do. The alternative would probably be overengineered if we would make the rows into own classes itself and they must decide on how and if they should be displayed in boldness. This also would not make this flag go away.

Anyhow, I suggest renaming this parameter to something like `highlightRowPrincipalName` or `principalNameHighlight` or something similar to reflect on what it is actually doing.